kodilla-testing/src/test/java/com/kodilla/testing/library/BookDirectoryTestSuite.java

```diff
@@ -85,32 +86,51 @@ public void testListBooksWithConditionFragmentShorterThan3() {
 
     @Test
     public void testListBooksInHandsOfWhenUserHave0Book() {
-        // Given
-        LibraryUser libraryUser = new LibraryUser("Jan", "Kowalczyk", "123345679");
+        //Given
+        LibraryUser libraryUserJan = new LibraryUser("Jan", "Kowalczyk", "123345679");
         LibraryDatabase libraryDatabaseMock = mock(LibraryDatabase.class);
```

Comment:
generalnie to powinno być pole klasy testowej, masz we wszystkich metodach testowych tego samego mocka przecież